docs/content/foundations/color.mdx

```diff
@@ -4,9 +4,1354 @@ page_title: Color
+          hex_value: '#757575'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-gray-70'
+          hex_value: '#454545'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-gray-90'
+          hex_value: '#171717'
+          additional_example_classes: 'radius-md padding-05'
+        - sub_header: Basic Pallette
+        - class_name: 'text-teal'
+          hex_value: '#298085'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-cerulean'
+          hex_value: '#007bbd'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-cranberry'
+          hex_value: '#bb0e3d'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-navy'
+          hex_value: '#14315c'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-golden'
+          hex_value: '#947100'
+          additional_example_classes: 'radius-md padding-05'
+  - name: Background color
+    utility_module_name: background-color
+    mixins_and_functions:
+      intro: |
+        Utilities may be used in component Sass with utility mixins and do not require utilities class output to be enabled.
+        Highlighted tokens are the NCIDS design tokens that serve as the foundation for all design system styles.
+        **Some** tokens are also a documentation link. Consult the Design tokens section for more information on tokens and how to use them in NCIDS.
+      outtro: |
+        * Use single-quoted strings in utility mixins. Do not quote numbers or px values, with the exception of half `(05)` values like `'05'`, `'105'`, and `'205'` which should be treated as strings.
+        * String tokens for half values (05) and negative values (like `neg-1`)may also be written with their unquoted number token equivalent- `0.5`, `1.5`, `2.5`, `2.5`, `-1`, `-1.5`, etc
+        * If multiple utilities share the same prefix (ex: `text-` or `.flex-`) or a single utility accepts multiple kinds of values (ex: .border accepts both border weight and border color), their shared mixin can accept multiple comma-separated values: `u-text('primary-darkest', 'no-underline', 'bold')` or `u-border-top(2px, 'accent-warm')`.
+        * Output any utility mixin as !important by appending !important to the mixin’s argument list: `u-bg('red-warm-50v', !important)`
+    utility_class_display_component: 'ColorDemonstration'
+    utility_class_display_params:
+      title_bar:
+        utility_name: 'Background color'
+        class_name: 'bg-color'
+        utility_states: 'hover'
+      classes:
+        - sub_header: 'State colors'
+        - class_name: 'bg-error-lighter'
+          hex_value: '#fde2ea'
+          additional_example_classes: 'font-mono-xs'
+        - class_name: 'bg-error-light'
+          hex_value: '#f27da2'
+          additional_example_classes: 'font-mono-xs'
```

Comment:
2 things for the same thing:
1. `additional_example_classes` should not be defined on every class if it is the same. You should handle that at the `utility_class_display_params` level.
2. Should you even make this optional and not just set the table body to this font.